README.md

```diff
@@ -852,6 +852,26 @@ are rejected over HTTP GET connections.  Derive from `GraphQLHttpMiddleware<T>`
 
 As would be expected, subscription requests are only allowed over WebSocket channels.
 
+### Excessive `OperationCanceledException`s
+
+When hosting a WebSockets endpoint, it may be common for clients to simply disconnect rather
+than gracefully terminating the connection -- most specifically when the client is a web browser.
```

Comment:
No; it represents an em dash. 

https://www.thepunctuationguide.com/em-dash.html

I can put the Unicode symbol in here. (Normally I just use two dashes for an em dash.)

Reply:
It's just that an em dash doesn't look much different than a regular dash with fixed-width type. So for clarity in a mono space editor, two dashes is more readable as an em dash than a Unicode em dash.

Reply:
OK. I would switch to real em dash — instead of - or -- since GitHub renders md files well. I tried to type — in VS and agree that it only a bit differs from -.